boards.go

```diff
@@ -57,6 +57,90 @@ func (b BoardList) String() string {
 	return Stringify(b)
 }
 
+// CreateIssueBoardOptions represents the available CreateIssueBoard() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/boards.html#create-a-board-starter
+type CreateIssueBoardOptions struct {
+	Name *string `url:"name" json:"name"`
+}
+
+// CreateIssueBoard creates a new issue board.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/boards.html#create-a-board-starter
+func (s *IssueBoardsService) CreateIssueBoard(pid interface{}, opt *CreateIssueBoardOptions, options ...OptionFunc) (*IssueBoard, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/boards", url.QueryEscape(project))
+
+	req, err := s.client.NewRequest("POST", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	board := new(IssueBoard)
+	resp, err := s.client.Do(req, board)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return board, resp, err
+}
+
+// UpdateIssueBoardOptions represents the available UpdateIssueBoard() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/boards.html#update-a-board-starter
+type UpdateIssueBoardOptions struct {
+	Name        *string `url:"name" json:"name"`
+	AssigneeID  *int    `url:"assignee_id" json:"assignee_id"`
+	MilestoneID *int    `url:"milestone_id" json:"milestone_id"`
+	Labels      *string `url:"labels" json:"labels"`
```

Comment:
According to the [API docs](https://docs.gitlab.com/ee/api/boards.html#update-a-board-starter), this endpoint takes in a "comma separated list of label names", so I just made this field a string as well. I couldn't find any other examples of how this is handled previously as I believe most API endpoints that operate on lists of things actually take in an array as well.

Do you want me to keep this as-is or make the model take in an array and transform it internally in the `UpdateIssueBoard` func?

Reply:
You can use the `Labels` type for that: https://github.com/xanzy/go-gitlab/blob/eca1beae81589cf0cfc27500316185b52a44eb07/issues.go#L102

Reply:
Ah, sweet - adjusted the code now. Thanks!